newIDE/app/src/InstancesEditor/TileSetVisualizer.js

```diff
@@ -374,11 +387,33 @@ const TileSetVisualizer = ({
         isBadlyConfigured ||
         !clickStartCoordinates ||
         !displayedTileSize ||
-        (!allowMultipleSelection && !allowRectangleSelection) ||
-        event.pointerType === 'touch'
+        (!allowMultipleSelection && !allowRectangleSelection)
       ) {
         return;
       }
+
+      let startCoordinates = clickStartCoordinates;
+
+      const isTouchDevice = event.pointerType === 'touch';
+
+      if (isTouchDevice) {
+        if (!touchStartCoordinates) return;
+
+        if (isLongTouch) {
```

Comment:
Add a comment explaining what this does:

"Distinguish between a long touch (to multi select tiles) and a scroll"
(for example)